go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/bgentry/go-netrc v0.0.0-20140422174119-9fd32a8b3d3d
 	github.com/hashicorp/terraform-plugin-sdk v1.15.0
 	github.com/mitchellh/go-homedir v1.1.0
-	github.com/signalfx/signalfx-go v1.8.3
+	github.com/signalfx/signalfx-go v1.8.5-0.20220125181716-88f2f325c069
```

Comment:
I did a proper release of 1.8.5 in signalfx-go, so can you change it here to use the new tag?

Reply:
Thanks! Done.